src/components/Header/Header.jsx

```diff
@@ -137,13 +137,17 @@ class Header extends React.Component {
       </VerticalAlignToolbar>
     );
 
-    const SearchGroup = () => (
+    const SearchGroup = ({ small }) => (
```

Comment:
are you actually using this prop?

Reply:
yes. SearchGroup is a parent of SearchForm. I use it in SearchForm.jsx

Reply:
or wait let me double check....

Reply:
ok it goes Redux Store -> Header -> SearchGroup > SearchForm

Reply:
this is probably bad. I should not do this :) I'm gonna get the prop in SearchForm directly.